Tiltfile

```diff
@@ -364,10 +500,17 @@ def deploy_observability():
 def prepare_all():
     tools_arg = "--tools kustomize,envsubst "
     tilt_settings_file_arg = "--tilt-settings-file " + tilt_file
+    kustomize_substitutions = settings.get("kustomize_substitutions", {})
```

Comment:
Can we just pass in an array like that into tilt-prepare? I don't see a flag for it 

(also we seem to set them to the env in l.515)

Reply:
Also left over from previous impementation, fixed now.